WelcomePage/WelcomePage.js

```diff
@@ -0,0 +1,15 @@
+const inpName = document.getElementsByClassName('inpName')[0];
+const btnLogin = document.getElementsByClassName('btnLogin')[0];
+
+btnLogin.onclik = welcome;
```

Comment:
I think it is onclik not onclick
I am not sure if welcome needs () or not

Reply:
Oh, yes it's onclick not onclik, sory
I will rewrite it.
No, we don't need to put () when
 invok function